orangecontrib/network/widgets/OWNxHist.py

```diff
@@ -208,11 +208,11 @@ def generateGraph(self, N_changed=False):
             graph.add_nodes_from(range(self.matrix.shape[0]))
             matrix = self.matrix
 
-            if hasattr(self.matrix, "items") and self.matrix.items is not None:
-                if isinstance(self.matrix.items, Orange.data.Table):
-                    graph.set_items(self.matrix.items)
+            if hasattr(self.matrix, "row_items") and self.matrix.row_items is not None:
```

Comment:
Has no effect because the place was incorrect, and I've only glanced it before. :flushed:  I hope the general method _is_ correct as it was applied in https://github.com/biolab/orange3-network/pull/2. Where there is a will, there is a way. :sunny: